cachi2/core/package_managers/yarn/main.py

```diff
@@ -84,11 +86,13 @@ def _fetch_dependencies(source_dir: RootedPath, output_dir: RootedPath) -> None:
 
     :param source_dir: the directory in which the yarn command will be called.
     :param output_dir: the directory where the yarn dependencies will be downloaded to.
-    :raises SubprocessCallError: if the 'yarn install' command fails.
+    :raises CalledProcessError: if the 'yarn install' command fails.
     """
-    # the output_dir is where the globalFolder must be set to
-    # the yarn commands can be called by using the core.utils.run_cmd function
-    pass
+    args = ["install", "--mode", "skip-build"]
+    try:
+        run_yarn_cmd(args, source_dir, {"YARN_GLOBAL_FOLDER": str(output_dir)})
```

Comment:
The GLOBAL_FOLDER should be output_dir/deps/yarn or output_dir/deps/yarnberry

It's probably fine to call it yarn, when (if) the time comes to implement Yarn v1 we can download those to deps/yarn-classic